dev/buildtool/inspection_commands.py

```diff
@@ -296,7 +296,7 @@ def join_results(thread_results, result_map):
 
   def ingest_bom_list(self, bom_list):
     """Ingest each of the boms."""
-    max_threads = 1 if self.options.one_at_a_time else 64
+    max_threads = 1 if self.options.one_at_a_time else 2
```

Comment:
curious about this "one_at_a_time" flag -- can we remove it or are there conditions under which it's set?

Reply:
Yeah, this seems like another flag we could delete; it looks like it's referenced a few other places and I didn't want to get into too much of a refactor here given that it wasn't causing any issues.

Reply:
🚫 🐜 🙏 ✅ 💯